pkg/server/server.go

```diff
@@ -203,7 +206,7 @@ func (s *ImmuServer) Stop() error {
 }
 
 func (s *ImmuServer) Login(ctx context.Context, r *schema.LoginRequest) (*schema.LoginResponse, error) {
-	if !s.Options.Auth && !auth.IsAdminClient(ctx) {
```

Comment:
old options.Auth is useless now?

Reply:
@mmeloni No, it is not useless: `auth.AuthEnabled` is set with the value from `s.Options.Auth` during server start. The "dynamic" update (done from `immuadmin`, while server is running is updating both, so it doesn't really matter which one of them is checked here).